python/tvm/tir/transform/transform.py

```diff
@@ -1040,3 +1040,13 @@ def InstallDebugSpans():
         The result pass
     """
     return _ffi_api.InstallDebugSpans()  # type: ignore
+
+
+def DefaultSchedule():
+    """Default schedule for PrimFuncs.
```

Comment:
I’m thinking about the performance. If we don’t intend to guarantee a decent performance, we probably should state in the documents that this pass only intends to make the IRModule able to be built, and has no guarantee on the performance.

BTW could we give the pass more documents? The PR description looks very good to serve as document.

Reply:
Thanks for the advice! Added more document according to PR description and the intention of this PR.